aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -182,6 +182,23 @@ module aptos_framework::transaction_context {
         payload.entry_function_payload
     }
 
+    #[test_only]
+    public fun new_entry_function_apyload(
```

Comment:
```suggestion
    public fun new_entry_function_payload(
```

Reply:
why not just inlining the constructor?

Reply:
what do you mean by inlining the constructor?

It's only possible to build the struct in the same module as where it's defined